conda/post-link.sh

```diff
@@ -3,4 +3,5 @@
 $PREFIX/bin/pip install \
 'rocksdict>=0.3,<0.4' \
 'protobuf>=5.27.2,<6.0' \
-'influxdb3-python>=0.7,<1.0'
+'influxdb3-python>=0.7,<1.0' \
+'pyiceberg[pyarrow,glue]>=0.7,<0.8'
```

Comment:
Would moving this to a `conda/requirements.txt` make this clearer ?

Then the `post-link.sh` can include `pip install -r conda/requirements.txt`